flutter_local_notifications/lib/src/platform_specifics/android/notification_details.dart

```diff
@@ -93,6 +95,13 @@ class AndroidNotificationAction {
   /// Set whether the notification should be canceled when this action is
   /// selected.
   final bool cancelNotification;
+
+  /// Sets the SemanticAction for this Action. A SemanticAction denotes what an
```

Comment:
A few issues on the API docs

- Can you follow [this guide](https://dart.dev/effective-dart/documentation)? Stand out are making sure first sentence is its own paragraph based on [this rule](https://github.com/MaikuB/flutter_local_notifications/blob/master/CONTRIBUTING.md)
- Can you look at making this a better/more explicit explanation on what this would do? On its own the first sentence `Sets the SemanticAction for this Action` doesn't make sense and reads awkward with how it mentions action twice. I'd suggest looking at the official [Android docs](https://developer.android.com/reference/androidx/core/app/NotificationCompat.Action.SemanticAction) for inspiration. As an example, "Provides meaning for this action." is a better explanation in plain English.
- Use "action" over "Action"